.github/workflows/bot-ts.yml

```
@@ -0,0 +1,190 @@
+concurrency:
+  group: "bot-ts @ ${{ github.event_name }} @ ${{ github.ref_name }}"
+  cancel-in-progress: true
+
+jobs:
+  build_per_arch:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: amd64
+            runner: ${{ env.RUNNER_AMD64 }}
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)